### ISSUE TITLE
Airvolute DroneCore 2.0 carrier board

### ISSUE THREAD

**Issue by B4njoS** [airvolute/dcs-deploy#15]
Hey!
Our company just purchased the new Airvolute DroneCore 2.0 carrier board with OrinNX, I have searched over your website and GitHub repo's to the flashing process and found nothing, can you provide me some instructions to the flashing process? send me a guide or something else..
We have no use for this board because we didn't find any flashing files and guide and I need it ASAP.
Regards.

**Comment by eMrazSVK** [comment]
Hello, you can use this branch https://github.com/airvolute/dcs-deploy/tree/jp_512_20_configs

Then, the steps are the same. You just specify config you want to use.

**Comment by B4njoS** [comment]
and still use the
"python3 dcs_deploy.py flash xavier_nx 51 1.2 nvme full" command to flash? i saw this branch but i thought it related to v1.2 carrier board

**Comment by B4njoS** [comment]
may you have JP5.1 for board v2? we have a lot of develop in JP5.1 and we want to stick to this version

**Comment by eMrazSVK** [comment]
No, you cannot use command where you specify jetson type as `xavier_nx` for Orin Nx, nor you can use `1.2` specifier for `2.0` board. 

You can refer to script `help` section by running `python dcs_deploy.py -h`. And then there is a help message for each command that the script support. You can use `list` argument to see all available configurations. There is JP 5.1 also for Orin and 2.0 board.

**Comment by B4njoS** [comment]
in the same script under the jp_512_20_configs branch there is also JP5.1?
how can i know what command will flash the device? can someone provide it?
